client/components/companies/CompanyItem.tsx

```diff
@@ -41,9 +44,15 @@ export default function CompanyItem() {
           alt="company"
         />
       </Container>
-      <Text as="i">Open Jobs: 10</Text>
+      <Text as="i">
+        {t('components.companies.company_item.open_jobs', {
+          count: 1 /* Placeholder count */,
+        })}
+      </Text>
       <br />
-      <ButtonGray>View More</ButtonGray>
+      <ButtonGray>
+        {t('components.companies.company_item.button_text')}
```

Comment:
Could you help me understand where it pulls the string from this to translate it? I can't seem to find the connection from `'components.companies.company_item.button_text` -> "View More"

Reply:
Scratch that. I found it at the bottom of the PR 🤙🤙